code/modules/surgery/organs/organ_external.dm

```diff
@@ -552,6 +523,63 @@ Note that amputating the affected organ does in fact remove the infection from t
 			qdel(src) // If you flashed away to ashes, YOU FLASHED AWAY TO ASHES
 			return null
 
+/obj/item/organ/external/proc/disembowel(spillage_zone = "chest")
+	if(!owner)
+		return
+
+	var/mob/living/carbon/C = owner
+
+	if(!hasorgans(C))
+		return
+
+	var/organ_spilled = FALSE
+	var/turf/T = get_turf(C)
+	C.add_splatter_floor(T)
+	playsound(get_turf(C), 'sound/effects/splat.ogg', 25, 1)
+	for(var/X in C.internal_organs)
+		var/obj/item/organ/O = X
+		var/org_zone = check_zone(O.parent_organ)
+		if(org_zone == spillage_zone)
+			O.remove(C)
+			O.forceMove(T)
+			organ_spilled = TRUE
+
+	if(organ_spilled)
+		C.visible_message("<span class='danger'><B>[C]'s internal organs spill out onto the floor!</B></span>")
+	return TRUE
+
+/obj/item/organ/external/chest/droplimb()
+	if(disembowel())
+		return TRUE
+
+/obj/item/organ/external/groin/droplimb()
+	if(disembowel("groin"))
+		return TRUE
+
+
+
+/obj/item/organ/external/attackby(obj/item/I, mob/user, params)
+	if(I.sharp)
```

Comment:
Should we be able to cut through synthetic torsos/groins/heads like hot knives through butter?

Reply:
well, I mean, they seem to be made of aluminum anyway....

Reply:
I like to pretend all this still means something *shrug*. 
Whatever way it goes I suuppose.

Reply:
Yes.